app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -310,37 +310,6 @@ def hosts(extra_attributes = {})
       attributes.merge!(extra_attributes)
     end
 
-    def vms(extra_attributes = {})
```

Comment:
@borod108 do you have specs for the reconnect in RHEV? We are trying to unify the behavior.

Reply:
@Ladas I do not think so, I will add some, can you please explain the functionality?

Reply:
@borod108 you had similar BZ for RHEV https://bugzilla.redhat.com/show_bug.cgi?id=1536406
So just that the archived Vm is reused, when deleting/re-adding provider.

Reply:
@pkliczewski irc you wrote a test for that?

Reply:
@borod108 Yes, I wrote a test for reconnecting a host and vm